saq/queue/postgres_migrations.py

```diff
@@ -43,4 +43,15 @@ def get_migrations(
                 ).format(stats_table=stats_table),
             ],
         ),
+        (
+            2,
+            [
+                SQL(
+                    dedent("""
+ALTER TABLE {stats_table} ADD COLUMN IF NOT EXISTS metadata JSONB;
+ALTER TABLE {stats_table} ADD COLUMN IF NOT EXISTS queue_key TEXT;
```

Comment:
looks like we're missing an index on expire_at, can you check the queries and see if that's the index that makes sense?